### DISCUSSION TITLE
Winter CMS htm page save issue

### DISCUSSION THREAD

**Discussion by waleedbajwa1980** [wintercms/winter#1127] [Q&A] [answered]
Hello dear members,

There is an urgent issue that needs to be resolved. infact i shifted the CMS from one domain to another domain. Now i logged into backend but when i click save while editing any .htm file it says

"Call to undefined method System\Models\LogSetting::hasDatabaseTable()" on line 71 of /home/jrnpmfgi/poc.matrixdas.com/vendor/laravel/framework/src/Illuminate/Support/Traits/ForwardsCalls.php

Moreover on front end the home.htm page after login is not being found by CMS although it does exists on backend. SO after login system send me back to login page saying page not found, i have checked the routes they are correct.

**Comment by LukeTowers** [comment]
Sounds like your vendor folder is out of date with the modules folder. How did you move it from one domain to another?

**Reply by waleedbajwa1980** [reply]
Many thanks for caring , i had just made zip file from previous hosting and extracted to current hosting, changed config/database.php and .env file. Cleared app cache, routes cache, config cache. Setup DB etc.

Also performed composer update.

![wintercms_error](https://github.com/wintercms/winter/assets/161030502/e8bef9e1-ec11-48ab-9ae9-1a5105ae4d74)

**Reply by LukeTowers** [reply, marked as answer]
Does your copy of this file match the one on github: https://github.com/wintercms/storm/blob/cf8fd3468e6cefdd6c0ff72a93de08ebd4800abb/src/Database/Model.php#L92

**Comment by waleedbajwa1980** [comment]
Thanks Luke, i just replaced above file with one at /winter/storm/src/Database/Model.php, htm templates being to save now.

**Reply by LukeTowers** [reply]
Please run `composer update` in your project folder instead, you are going to have other much more difficult issues to solve if your dependencies don't match each other.

**Reply by waleedbajwa1980** [reply]
Yes i did that also. All is working fine now.
